buildSrc/src/main/kotlin/DokkaExts.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
In the following PR, let's change this to `https` throughout the project, and in the IDEA's header's template.